### DISCUSSION TITLE
Add unlit soul torch

### DISCUSSION THREAD

**Discussion by legosteenjaap** [realguyman/totally_lit#11] [Ideas]

**Comment by realguyman** [comment]
I'm thinking I can replace Minecraft's soul torch recipe to craft an unlit variant. By default soul torches will be more resistant to water and last an infinite amount of time. It'd make sense to have a campfire recipe for it, too.

**Comment by Andrew6rant** [comment]
If you do decide to add Unlit Soul Torches, I think it would be very cool to also make mob death be a trigger that lights them

**Comment by realguyman** [comment]
Unlit soul torches have been added as of https://github.com/realguyman/totally_lit/commit/3639b7b11897e8da65498949e897ccdb501170f3. Scheduled to be part of 0.11.0 release.
